webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue

```diff
@@ -852,6 +926,10 @@ export default {
     },
     getString(body) {
       return new DOMParser().parseFromString(body, 'text/html').documentElement.textContent.replace(/&nbsp;/g, '').trim();
+    },
+    updateDraftVisibility(){
```

Comment:
you should use this.$newsServices.updateNews exactly when we modify another news field